api/envoy/config/core/v3/protocol.proto

```diff
@@ -102,6 +102,10 @@ message QuicProtocolOptions {
   // A comma-separated list of strings representing QUIC client connection options defined in
   // `QUICHE <https://github.com/google/quiche/blob/main/quiche/quic/core/crypto/crypto_protocol.h>`_ and to be sent by upstream connections.
   string client_connection_options = 7;
+
+  // The duration that a QUIC connection stays idle before it closes itself. If this field is not present, QUIC default will be applied.
+  google.protobuf.UInt32Value idle_timeout_seconds = 8
```

Comment:
Per API style this should be google.protobuf.Duration instead of integer in seconds.

Reply:
done